pkg/config/usagereportingconfig.go

```diff
@@ -74,30 +74,36 @@ func NewUsageReporting() UsageReportingConfig {
 		Offline:           false,
 		Schedule:          "@hourly",
 		reportSchedule:    "@monthly",
-		reportGranularity: 3600000,
+		reportGranularity: 900000,
 		qaVars:            false,
 	}
 }
 
+func (u *UsageReportingConfiguration) deprecationWarning(old string, new string) {
```

Comment:
Nice... i should go back... when you updated the go template stuff, did you also create a deprecation warning func?  if not, maybe I should go back in and create one.

Reply:
I didn't that is a good idea.  I think this should be moved to the logging package? As it can be reused elsewhere.

Reply:
even better... if you do it, i'll use it... otherwise, i can and move it to the logging package and change this one... either or.

Reply:
Let me see if I can add it here, no probs